docs/ko/reference/array/unzipWith.md

```diff
@@ -0,0 +1,32 @@
+# unzipWith
+
+입력된 배열을 분리하거나 변환하여 새로운 배열로 반환하는 함수입니다. 이 함수는 중첩된 배열을 받아서 각 요소들을 분리하여 새로운 배열을 만들거나, 선택적으로 제공된 함수를 이용해 변환된 값으로 새로운 배열을 생성합니다.
+
+## 인터페이스
+
+```typescript
+function unzipWith<T>(target: readonly T[][] | null | undefined): T[][];
+function unzipWith<T, R>(target: readonly T[][] | null | undefined, iteratee: (...args: T[]) => R): R[];
```

Comment:
```suggestion
function unzipWith<T, R>(target: readonly T[][], iteratee: (...args: T[]) => R): R[];
```

We should probably only support cases where `iteratee` is present, since there's no real use for `unzipWith` otherwise.

Reply:
Thank you for checking!